<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>照片抠图及证件照制作</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0,4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,4,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>原图</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="originLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>不抠图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>抠图</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,4,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>抠图后</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="removeBGLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="previewPushButton">
          <property name="text">
           <string>查看抠图</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="saveResultpushButton">
          <property name="text">
           <string>保存抠图</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>底色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="blueRadioButton">
          <property name="text">
           <string>蓝色</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">colorButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="redRadioButton">
          <property name="text">
           <string>红色</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">colorButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>尺寸</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oneRadioButton">
          <property name="text">
           <string>一寸 x 五寸</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sizeButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string>二寸 x 五寸</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sizeButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oneRadioButton_2">
          <property name="text">
           <string>一寸 x 六寸</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sizeButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string>二寸 x 六寸</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sizeButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="oneRadioButton_3">
          <property name="text">
           <string>5 寸混排</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sizeButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_6">
          <property name="text">
           <string>6 寸混排</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">sizeButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="5,0">
      <item>
       <widget class="QLabel" name="IndesignLabel">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>保存排版</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="openPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>打开保存</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>说明</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="declareAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="helpAction">
   <property name="text">
    <string>操作说明</string>
   </property>
  </action>
  <action name="declareAction">
   <property name="text">
    <string>声明</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>colorButtonGroup</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>colorSelectAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectPhotoAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeBackgroundAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>616</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previewPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>amplifyAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>632</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveResultpushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveRemoveResultAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveDesignResultAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeButtonGroup</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sizeSelectAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>helpAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>declareAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>declareAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openSaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>753</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>noRemoveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectPhotoAction()</slot>
  <slot>removeBackgroundAction()</slot>
  <slot>amplifyAction()</slot>
  <slot>saveRemoveResultAction()</slot>
  <slot>saveDesignResultAction()</slot>
  <slot>printPhotoAction()</slot>
  <slot>colorSelectAction()</slot>
  <slot>sizeSelectAction()</slot>
  <slot>helpAction()</slot>
  <slot>aboutAction()</slot>
  <slot>declareAction()</slot>
  <slot>openSaveAction()</slot>
  <slot>noRemoveAction()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="sizeButtonGroup"/>
  <buttongroup name="colorButtonGroup"/>
 </buttongroups>
</ui>
